bravado/response.py

```diff
@@ -41,8 +40,18 @@ def __init__(self, incoming_response, swagger_result, is_fallback_result, elapse
 
     @property
     def status_code(self):
+        if not self.incoming_response:
+            raise ValueError('No incoming_response present')
```

Comment:
Feel safer :)
How about having `incoming_response` as a property that raises the exception or having a small helper that asserts the `incoming_response` presence? we can avoid to do the same check all the time